azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/CSAgentPoolProfile.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.compute;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Attachable;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * An client-side representation for a container service agent pool.
+ */
+@Fluent
+public interface CSAgentPoolProfile extends

```

Comment:
not sure about using a poorly known acronym here. I think it'd be better to spell this out `ContainerServicesAgentPoolProfile`

Also - is the word `Profile` needed?  Couldn't it simply say: `ContainerServicesAgentPool`?